src/components/inline-link/inline-link.module.css

```diff
@@ -29,5 +29,9 @@ ones set in the surrounding text.
 
   &:visited {
     color: var(--token-color-foreground-action-visited);
+
+    &:hover {
+      color: var(--token-color-palette-purple-400);
```

Comment:
@ashleemboyer there is a specific token for this: `--token-color-foreground-action-hover` https://github.com/hashicorp/design-system/blob/0d950ac091cf50d6cf06ae67a4435ddf07d050b7/packages/tokens/dist/devdot/css/tokens.css#L122

Reply:
Sure 👍

Reply:
@ashleemboyer FYI https://github.com/hashicorp/design-system/pull/83 
I suggest to wait and use directly that token (we should be releasing it very soon)

Reply:
Will do!